x/programs/cmd/simulator/vm/actions/program_execute.go

```diff
@@ -60,80 +66,35 @@ func (t *ProgramExecute) Execute(
 	_ int64,
 	actor codec.Address,
 	_ ids.ID,
-) (success bool, computeUnits uint64, output []byte, err error) {
```

Comment:
The reason of this is https://github.com/ava-labs/hypersdk/pull/858/commits/b67c5f07936fc758fc78b90f702449e3e7f29185